falco/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: falco
-version: 1.7.9
+version: 1.8.0
```

this is just a bug fix you can bump only the patch version 
```suggestion
version: 1.7.10
```

---

My fault, I read the `CHANGELOG.md` file and found that the last bit of the version did not exceed 10, so it was updated to 1.8.0. I should communicate with you first.